src/tests/test_ssh_client.c

```diff
@@ -25,6 +25,9 @@
 #error "The path to the ssh authorizedkeys helper is not defined"
 #endif /* SSH_CLIENT_DIR */
 
+#define GOTO_END(code)  {return_code = code;\
+                         goto end;}
+
```

Comment:
Hi,

thank you for the fix. I would prefer if you can drop this macro and just have the two lines in all the needed places in the code. In general we try to avoid such error handling macros.

bye,
Sumit

Reply:
ok, thanks review.i will fixed.